packages/text-bitmap/test/BitmapText.js

```diff
@@ -71,7 +71,7 @@ describe('PIXI.BitmapText', function ()
             font: this.font.font,
         });
 
-        expect(text.children.length).to.equal(4);
+        expect(text._activePagesMeshData[0].total).to.equal(4);
     });
```

Comment:
I think a test where you sinon-spy on the addChild/removeChild and verify it works as intended would finish this one solidly.

For example:
* BitmapFont(ALPHANUMERIC, fontSize: 8) (make sure it outputs only one page with small enough font size)
* BitmapText("HelloWorld")
* updateText() -> addChild 1 times (because only one page, right?)
* updateText() again -> addChild 0 times
* change text to "dlorWolleH"
* updateText() -> addChild/removeChild 0 times
* change text to ""
* updateText() -> removeChild 1 time

Reply:
Good shout! Will add this in later today 👍 
what is sinon-spy? (I'm test newbie, looking to improve!)

Reply:
See https://sinonjs.org/releases/v9.0.2/spies/ (sinon.spy())

We are using sinon-spy in our tests already: https://github.com/pixijs/pixi.js/blob/2daa8749845e23525f10607f402c434390bf4b43/packages/ticker/test/Ticker.js#L99 is an example.

Reply:
thanks, will take a look!